R/create.R

```diff
@@ -194,30 +193,36 @@ create_from_github <- function(repo_spec,
   }
 
   if (open) {
-    open_project(proj_get())
+    open_project(proj_get(), restore = old_project)
   }
+  invisible(TRUE)
+
 }
 
-open_project <- function(path, rstudio = NA) {
-  rproj_path <- rproj_path(path)
+## `rstudio` arg here is about whether to attempt a launch in RStudio
+## `rstudio` arg of `create_*()` functions is about whether to add .Rproj file
+open_project <- function(path, restore = NA, rstudio = NA) {
```

Comment:
The default for `restore` must be `NA`, because `NULL` is a valid value to restore, i.e. we could need to restore the state of "no active project".